packages/amplify-e2e-tests/src/cleanup-e2e-resources.ts

```diff
@@ -829,6 +905,13 @@ const cleanupAccount = async (account: AWSAccountInfo, accountIndex: number, fil
   const allResources = mergeResourcesByCCIJob(
     apps, stacks, buckets, orphanBuckets, orphanIamRoles, orphanPinpointApplications, orphanAppSyncApis
   );
+  // cleanup resources that are <unknown> but that are definitely amplify resources
```

Comment:
the purpose of the code below is to re-classify some specific resources as orphaned resources, so that they are not immune to deletion

Reply:
Would be helpful to have a comment that briefly describes what we're looking for. Something like "selects stacks that contain "test" or "amplify" within the name"

Reply:
updated